### PR TITLE
docs(README): fix the `git committer string` and `Configure git CLI` examples

### PR DESCRIPTION
**Pull request by maboloshi**
1. Fix the `git committer string` and `Configure git CLI` examples.
2. Formatting examples
3. Modify the get `<BOT USER ID>` comment and add a note that [octokit/request-action](https://github.com/octokit/request-action) can be used.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,23 +104,27 @@ jobs:
...
           git commit -m "Auto-generated changes"
           git push
 ```
 
-The `<BOT USER ID>` is the numeric user ID of the app's bot user, which can be found under `https://api.github.com/users/<app-slug>%5Bbot%5D`.
-For example, we can check at `https://api.github.com/users/dependabot%5Bbot%5D` to see the user ID of dependabot is 49699333.
+> [!TIP]
+> The `<BOT USER ID>` is the numeric user ID of the app's bot user, which can be found under `https://api.github.com/users/<app-slug>%5Bbot%5D`.
+> 
+> For example, we can check at `https://api.github.com/users/dependabot%5Bbot%5D` to see the user ID of dependabot is 49699333.
```
Comment (parkerbxyz) [inline README.md:125]:
```suggestion
> For example, we can check at `https://api.github.com/users/dependabot[bot]` to see the user ID of Dependabot is 49699333.
```

File: README.md [hunk 2]
```diff
@@ -104,23 +104,27 @@ jobs:
...
+>
+> Similarly to get the id, you can also use the [octokit/request-action](https://github.com/octokit/request-action).
```
Comment (parkerbxyz) [inline README.md:127]:
```suggestion
> Alternatively, you can use the [octokit/request-action](https://github.com/octokit/request-action) to get the ID.
```

**Review by parkerbxyz — COMMENTED** [review]
Thank you, @maboloshi! ✨ I left a couple of small suggested changes for your consideration.

**Review by gr2m — APPROVED** [review]
nice, thank you!

**Pull request merged**